modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -241,112 +352,18 @@ When using encrypted properties, it is especially important to **secure access t
 * ECB
 * OFB
 
-== Using the Extension in Anypoint Studio 7
-
-You can use this extension by adding it as a dependency in your Mule app.
-
-=== Installing the Extension
-
-. Open your Mule project in Anypoint Studio.
-
-. Go to the Mule Palette.
-
-. Select *Search in Exchange*, and search for the *Mule Secure Configuration Property Extension*.
-
-. Add the extension.
-
-=== Adding Secure Configuration Properties to your App
-
-. Go to your Mule app configuration file.
-
-. Select the `Global Elements` tab.
-
-. Click the `Create` button.
-
-. From the search bar, select `Secure Properties Config`.
-
-. Configure the global element with a `File` location, `Key`, `Algorithm`, and `Mode`:
-
-image::secure-configuration-properties-studio.png[config extension]
-
-[[secure_props_tool]]
-== Secure Properties Tool
-
-You can download the https://docs.mulesoft.com/downloads/mule-runtime/4.2/secure-properties-tool.jar[JAR file] for this tool so you can encrypt or decrypt single values, and complete files (both, YAML and Properties files).
-This tool supports three encryption modes: `string`, `file`, and `file-level`
-
-=== Encrypt or Decrypt Single values
-
-Use the `string` mode to encrypt or decrypt a single value provided as a parameter.
 
-[source,console.linenums]
-----
-java -jar secure-properties-tool.jar string <encrypt|decrypt> <algorithm> <mode> <key> <value>
-----
+== Frequently Asked Questions
```

Comment:
Consider adding some text here that ties this topic into the set and introduces how the audience should use it.
Maybe something like...:

"After you learn about, create, and use secure configuration properties files for a while, you might have additional questions about troubleshooting and maximizing their use. Following are some common questions and answers. Refer as well to the topics linked to in See Also.